frontend/package.json

```diff
@@ -33,6 +33,7 @@
     "postcss": "^8",
     "register-service-worker": "^1.7.2",
     "rrweb-snapshot": "^1.1.14",
+    "uuid": "^9.0.0",
```

Comment:
no need for exra dependency in case we do slice creation on the backend using conditions sent from the frontend